account_payment_term_partner_holiday/models/res_partner.py

```diff
@@ -34,7 +34,7 @@ def is_date_in_holiday(self, date):
                     res_date_to = fields.Date.from_string(
                         "%s-%s-%s" % (date.year, res.month_to, day_to)
                     )
-                except:
+                except BaseException:
```

Comment:
As we are changing this, could this maybe be a more specific exception? I guess this should probably come from the conversion from `string` to `Date`, so you probably want to catch a `ValueError`: https://docs.python.org/3/library/datetime.html#datetime.datetime.strptime